src/main/java/uk/gov/hmcts/divorce/common/service/task/SetSubmitAosState.java

```diff
@@ -41,7 +41,8 @@ public CaseDetails<CaseData, State> apply(CaseDetails<CaseData, State> caseDetai
             log.info("State not changed for AOS submission task for CaseID: {}", caseDetails.getId());
         }
 
-        if (caseData.getApplicant2().getLanguagePreferenceWelsh() == YesOrNo.YES) {
+        if (caseData.getApplicant2().getLanguagePreferenceWelsh() == YesOrNo.YES
```

Comment:
It is is usually preferable to use YES.equals(caseData.getApplicant2().getLanguagePreferenceWelsh()) in Java